packages/devtools_app/lib/src/shared/history_viewport.dart

```diff
@@ -37,21 +37,21 @@ class HistoryViewport<T> extends StatefulWidget {
   });
 
   final HistoryManager<T> history;
-  final String Function(T) generateTitle;
-  final List<Widget> controls;
   final Widget Function(BuildContext, T) contentBuilder;
-  final void Function(T, T) onChange;
+  final List<Widget>? controls;
+  final String Function(T)? generateTitle;
+  final void Function(T, T)? onChange;
   final bool historyEnabled;
-  final VoidCallback onTitleTap;
+  final VoidCallback? onTitleTap;
 
   @override
-  State<HistoryViewport<T>> createState() => _HistoryViewportState<T>();
+  State<HistoryViewport<T?>> createState() => _HistoryViewportState<T>();
 }
 
-class _HistoryViewportState<T> extends State<HistoryViewport<T>> {
-  TextStyle _titleStyle;
+class _HistoryViewportState<T> extends State<HistoryViewport<T?>> {
+  TextStyle? _titleStyle;
 
-  void _updateTitleStyle(TextStyle style) {
+  void _updateTitleStyle(TextStyle? style) {
```

Comment:
make this parameter non-nullable, then just unwrap `defaultTextStyle` `!` where it is defined.

Reply:
done